prow/plugins/label/label.go

```diff
@@ -146,11 +172,11 @@ func handle(gc githubClient, log *logrus.Entry, additionalLabels []string, e *gi
 		labelsToAdd         []string
 		labelsToRemove      []string
 	)
-
+	nonexistent = make([]string, 0)
 	// Get labels to add and labels to remove from regexp matches
-	labelsToAdd = append(getLabelsFromREMatches(labelMatches), getLabelsFromGenericMatches(customLabelMatches, additionalLabels)...)
-	labelsToRemove = append(getLabelsFromREMatches(removeLabelMatches), getLabelsFromGenericMatches(customRemoveLabelMatches, additionalLabels)...)
-
+	labelsToAdd = append(getLabelsFromREMatches(labelMatches), getLabelsFromGenericMatches(customLabelMatches, additionalLabels, &nonexistent)...)
+	labelsToRemove = append(getLabelsFromREMatches(removeLabelMatches), getLabelsFromGenericMatches(customRemoveLabelMatches, additionalLabels, &nonexistent)...)
+	nonexistent = unique(&nonexistent)
```

Comment:
Do this after we are done appending to nonexistent.
Consider using `unique` on `labelsToAdd` and `labelsToRemove` as well.

Reply:
Thank you. Will do so.